config/boot.rb

```diff
@@ -1,6 +1,3 @@
-require 'rubygems'
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
```

Comment:
What's the difference between dir and file for this line?

Reply:
this came with the upgrade

Reply:
Makes sense. Thanks.